src/main/kotlin/uk/gov/dluhc/printapi/service/temporarycertificate/PdfFactory.kt

```diff
@@ -16,6 +16,7 @@ class PdfFactory {
         PdfReader(ResourceUtils.getFile(templateDetails.path).inputStream()).use { reader ->
             val stamper = PdfStamper(reader, outputStream)
             stamper.cleanMetadata()
+            stamper.setFormFlattening(true)
```

Comment:
Flatten the form fields when the stamper is closed, meaning the resultant PDF has the field values, but they are no longer editable form fields (dont want the vc-admin or someone creating a PDF, then being able to edit it themselves afterwards!)

Reply:
Nice - wasn't aware of this feature; have updated my PR to include this change 👍

Reply:
Minor - perhaps add a comment to that effect?